src/GraphQL.DataLoader.Tests/SimpleDataLoaderTests.cs

```diff
@@ -59,7 +59,7 @@ public async Task Operation_Can_Be_Cancelled()
             mock.Setup(store => store.GetAllUsersAsync(cts.Token))
                 .Returns(async (CancellationToken ct) =>
                 {
-                    await Task.Delay(100);
+                    await Task.Delay(200);
                     ct.ThrowIfCancellationRequested();
```

Comment:
If you look further down, the cancellation happens after 5ms, and the delay here was 100ms, so really not sure why that test sometimes works and sometimes doesn't. There could be a more sinister bug here. Should we raise it as another issue to be looked at?

Reply:
Maybe the cancellation fires before work even runs, so the delay in `CancelAfter` should be a bit higher?

Reply:
I have been watching the history of this test for a long time. Something is really wrong with it. I assume that CI has strict restrictions on parallelism and because of this, the usual order of things is violated. Or an error somewhere else. In any case, you just need to repeat the build until the test is completed.

Reply:
> Maybe the cancellation fires before work even runs, so the delay in CancelAfter should be a bit higher?

Maybe, but now we need to finish this PR.